Updates old homepage_url key documentation (#4291)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/manifest/homepage_url/index.md

layout: "layouts/doc-post.njk"
title: "Manifest - Homepage Url"
date: 2013-05-12

> updated: 2022-11-14

Reviewer: ```suggestion
updated: 2023-01-04
```

updated: 2023-01-04
description: Reference documentation for the homepage_url property of manifest.json.
---

The URL of the homepage for this extension. The extensions management page (chrome://extensions)

> will contain a link to this URL. This field is particularly useful if you [host the extension on

Reviewer 2: Did MV3 introduce any changes around how an extension can be hosted/what the behaviour is in that case? Wondering if it makes sense to keep this line or not.

Reviewer: Question for Ian. He has the context on this PR.

Author: @oliverdunk My understanding is that this key only regards linking to the developer's home site, and not necessarily any explicit hosting of the extension.

Reviewer 2: Let's follow up on this if anything, I don't think it needs to block the PR 👍

> An optional Manifest key containing a String for a valid homepage URL. Developers may choose to set the extension's homepage to their personal or company's website. If the parameter is left undefined, the default homepage will be the extension's Chrome Web Store page listed on the extensions management page (chrome://extensions). This field is particularly useful if you [host the extension on your own site][1]. 

Reviewer: I'm working on style guidance for Extension-specific features. In general, manifest and string should NOT be capitalized. They are common nouns because they refer to classes of things rather than instances of things. It's the same as with 'I live in a city' vs. 'I live in Kansas City'.

Also, I should have made it clear yesterday that the kind of review we're asking for is a review of the technical accuracy of the PR. One of the writers will review for typos when you're done.

{
  // ...

>   "homepage_url": "https://validURL.com,",

Reviewer 2: Nit: Would it be better to use "https://path/to/homepage" to be consistent with https://developer.chrome.com/docs/extensions/mv3/manifest/ on `main`?

Alternatively we could use something like https://example.com (since that's a more formal looking example page) and make sure to update the manifest page too.

Reviewer: We should be example.com since the W3C has declared it off limits. Someone could register validURL.com and put up content we don't like.

layout: "layouts/doc-post.njk"
title: "Manifest - Homepage Url"
date: 2013-05-12
updated: 2022-11-14
description: Reference documentation for the homepage_url property of manifest.json.
---

> An optional Manifest key containing a String for a valid homepage url. Developers may choose to set the extension’s homepage to their personal or company’s webesite. If the parameter is left undefined, the default homepage will be the extension’s Chrome Web Store page listed on the extensions management page (chrome://extensions). This field is particularly useful if you [host the extension on
> your own site][1]. 

Reviewer 3: - Use proper casing on `URL`
- Fix typo in "website"
- Merge dangling line into a the paragraph

```suggestion
An optional Manifest key containing a String for a valid homepage URL. Developers may choose to set the extension's homepage to their personal or company's website. If the parameter is left undefined, the default homepage will be the extension's Chrome Web Store page listed on the extensions management page (chrome://extensions). This field is particularly useful if you [host the extension on your own site][1]. 
```

 ```json
{
  // ...

>       "homepage_url": "https://validURL.com,",

Reviewer 3: Remove extra space.

```suggestion
  "homepage_url": "https://validURL.com,",
```
